cmd/drone-docker/main.go

```diff
@@ -103,6 +103,11 @@ func main() {
 			Usage:  "docker daemon executes in debug mode",
 			EnvVar: "PLUGIN_DEBUG,DOCKER_LAUNCH_DEBUG",
 		},
+		cli.BoolTFlag{
+			Name:   "daemon.iptables",
+			Usage:  "docker daemon enable addition of iptables rules",
```

Comment:
also consider changing the description to make it clear that this flag disables iptable rules, instead of enabling